scripts/MCUXpresso_Config_Tools/windows-amd64/launch-MCUXpressoConfigTools.bat

```diff
@@ -0,0 +1,43 @@
+@echo off
+REM Script launch MCUXpresso Config Tools in OpenCMSIS generator mode. It is supported since MCUXpresso Config Tools v16
+
+REM get folder, where the script run
+set prjDir=%cd%
```

Comment:
The current working directory is not part of the specification, its behaviour is implementation defined.
To be consistently compliant with the specification I would recommend to use the information from the *.cbuild-gen-idx.yml file. 
For example the directory for the generated files can be found in the `output` node:
https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/YML-CBuild-Format.md#generator-information-files:~:text=csolution%20run%20command.-,output%3A,-Specifies%20the%20directory

Reply:
We decided to simplified the usage of config tools for now. I removed the implementation for legacy project, where the .mex file is in same folder as .csolution and .cproject and not in "output" directory.